lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_merge_strings_files.rb

```diff
@@ -4,7 +4,8 @@ class IosMergeStringsFilesAction < Action
       def self.run(params)
         UI.message "Merging strings files: #{params[:paths].inspect}"
 
-        duplicates = Fastlane::Helper::Ios::L10nHelper.merge_strings(paths: params[:paths], output_path: params[:destination])
+        destination = params[:destination] || params[:paths].keys.first # Use first path if destination not provided, to do an in-place merge.
+        duplicates = Fastlane::Helper::Ios::L10nHelper.merge_strings(paths: params[:paths], output_path: destination)
         duplicates.each do |dup_key|
           UI.important "Duplicate key found while merging the `.strings` files: `#{dup_key}`"
         end
```

Comment:
💡  Could be worth adding an additional `UI.important` help message at the end to suggest using that new prefix feature I just introduced in those cases:
```ruby
        UI.important "Tip: You might want to consider providing different prefixes in the `Hash` you used for the `paths:` parameter to avoid those key conflicts." unless duplicates.empty?
```
_(Not writing this as a GitHub suggestion because such a change in the `UI.important` messages will likely have to be mirrored in the specs testing the action in parallel to here)_

Reply:
Implemented in f9f1dafee6922d017a6e9af26938dc65d10da5f7